draft-ietf-mls-protocol.md

```diff
@@ -1479,7 +1573,8 @@ to be used for its own sending chain:
            application_secret_[sender]_[0]
 ~~~~~
 
-Note that [sender] represents the index of the member in the roster.
+Note that [sender] represents the index of the member among the
+leaves of the ratchet tree.
```

Comment:
Make clear if node or leaf index is meant here. E.g. "represents the index of the member among the leaves of the ratchet tree" --> "represents the leaf index of the ratchet tree leaf assigned to the sender"

Reply:
Done.